docs/tutorials/using_apis/Mbed_USB_WAV_Audio_Player/Mbed_USB_WAV_Audio_Player.md

```diff
@@ -0,0 +1,92 @@
+# Mbed USB WAV Audio Player
+This is a tutorial on how to put together a USB WAV audio player with Mbed. In general, the user will load a WAV file onto an SD card that will be connected to an Mbed board. The user can then play the WAV file from the board on a host computer over USB. Once connected, the Mbed board will act as a USB microphone. The sound that the microphone is "recording" is actually sound data read from the SD card and sent to the host computer. This allows the sound to be played on speakers, headphones, or recorded with a program like Audacity.
+
+### Libraries
+This tutorial uses several Mbed libraries:
+- USB (part of Mbed-OS)
+- [sd-driver](https://github.com/ARMmbed/sd-driver) (external library)
+- [AudioPlayer](https://github.com/c1728p9/AudioPlayer) (external library)
+
+The USBAudio class of USB library is used to send audio data from the Mbed board to the host PC. The SDBlockDevice and FATFileSystem classes of sd-driver library are used so that WAV files can be stored on an SD card and accessed by the Mbed board. Lastly, the AudioPlayer and WaveAudioStream classes of the AudioPlayer library are used to access the audio data from the WAV file on the SD card.
+
+### What You Need
+- Mbed Enabled board
+- SD card
+- SD card shield (if board does not already have an SD card port)
+- USB cable
+
+### Setup
+The following steps demonstrate the setup and use of the Mbed WAV Audio Player:
+1. Create a new Mbed project.   
+`mbed new <project name>`
+2. Add the [sd-driver](https://github.com/ARMmbed/sd-driver) and [AudioPlayer](https://github.com/c1728p9/AudioPlayer) libraries into the new Mbed project directory.   
+`mbed add https://github.com/ARMmbed/sd-driver`    
+`mbed add https://github.com/c1728p9/AudioPlayer`
+3. Copy and paste the below example code into *main.cpp*.
+4. Make sure that the SPI pins for the SDBlockDevice object are updated and correct for your board.
```

Comment:
"SPI pins for the SDBlockDevice" .. A reference to this in the supplied example will help to point where/what to check for SPI pins..